code/modules/food_and_drinks/drinks/drinks/cans.dm

```diff
@@ -1,18 +1,30 @@
 /obj/item/reagent_containers/food/drinks/cans
-	var/canopened = 0
+	var/canopened = FALSE
 	var/is_glass = 0
 	var/is_plastic = 0
+	var/shaken = 0
+	var/bursting = FALSE
```

Comment:
`can_burst` here? Bit more descriptive variable name will atleast help. `bursting` reads as if it's already bursting. AKA if an effect is playing or such but here it means that it can burst once the roll happens